src/main/java/uk/gov/justice/hmpps/prison/service/SearchOffenderService.java

```diff
@@ -51,7 +51,7 @@ public Page<OffenderBooking> findOffenders(final SearchOffenderRequest request)
         Objects.requireNonNull(request.getLocationPrefix(), "locationPrefix is a required parameter");
         log.info("Searching for offenders, criteria: {}", request);
 
-        final Set<String> caseloads = securityUtils.isOverrideRole() ? Set.of() : userService.getCaseLoadIds(request.getUsername());
+        final Set<String> caseloads = userService.getCaseLoadIds(request.getUsername());
```

Comment:
5 clients call the endpoint (/api/locations/description/{locationPrefix}/inmates) that calls this method.  Only categorisation-tool client has the SYSTEM_USER role, and it has not made a call to this endpoint since Jan 2023